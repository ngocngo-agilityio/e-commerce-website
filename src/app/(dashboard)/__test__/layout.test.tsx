import { render } from '@testing-library/react';

// Components
import DashboardLayout from '../layout';

jest.mock('@components', () => ({
  Footer: () => <div>Footer</div>,
}));

jest.mock('@sections', () => ({
  Header: () => <div>Header</div>,
}));

describe('DashboardLayout', () => {
  it('should match snapshot', () => {
    const { container } = render(
      <DashboardLayout>
        <div>Children component</div>
      </DashboardLayout>,
    );

    expect(container).toMatchSnapshot();
  });
});
