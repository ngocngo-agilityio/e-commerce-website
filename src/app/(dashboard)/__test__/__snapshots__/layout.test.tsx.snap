// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DashboardLayout should match snapshot 1`] = `
<div>
  <div>
    Header
  </div>
  <div
    class="css-qppafm"
  >
    <div>
      Children component
    </div>
  </div>
  <div>
    Footer
  </div>
</div>
`;
